source/customization.rst

```diff
@@ -1,32 +1,49 @@
 .. _customization:
 
-Customization
-=============
+Customization (or the many OOD environment variables)
+=====================================================
 
-Announcements
----------------
+Setting the PUN Environment With ``nginx_stage.yml``
+----------------------------------------------------
 
-.. attention:: TODO
+The configuration file ``nginx_stage.yml`` is the best way to set the environment which the PUN will use. Defining the mapping ``pun_custom_env`` allows setting Open OnDemand specific variables and is used for branding, message of the day setting, quota warnings, whitelists and other uses. For environment variables that OOD applications may need, are already in the environment and should be exposed to OOD (``$PATH``, ``$LD_LIBRARY_PATH``, etc...) define the sequence ``pun_custom_env_declarations``.
+
+An example of both of these uses may be found in `nginx_stage_example.yml <https://github.com/OSC/ondemand/blob/d85a3982d69746144d12bb808d2419b42ccc97a1/nginx_stage/share/nginx_stage_example.yml#L26-L43>`__
+
+Announcements and Message of the Day (MOTD)
+-------------------------------------------
+
+Two hooks for customizing the Dashboard are Announcements and MOTD. Announcements get the classes ``alert alert-warning`` and appear above the ``$OOD_DASHBOARD_LOGO`` . Announcement files are expected to be found at: ``/etc/ood/config/announcement.(md|yml)`` or ``/etc/ood/config/announcements.d/any_file_name.(md|yml)``. To display a MOTD file on the Dashboard ensure that the environment variables ``$MOTD_PATH`` and ``$MOTD_FORMAT`` are set like so:
```

Comment:
>appear above the ``$OOD_DASHBOARD_LOGO``

Maybe just "appear at the top of the page" - becuas announcements appear on the index page and also on the interactive app web form pages and the list of current interactive sessions and the developer views etc.